### ISSUE TITLE
itbit_btc_usd doesn't allow for anything less than 0.01 btc volume

### ISSUE THREAD

**Issue by itali43** [garethdmm/gryphon#11]
Sorry in advance, should probably be a pull request..

Found that this code in itbit's place_order function makes it so that anything below 3 decimal places in order volume comes in as 0.00.

Present Code:
```
        # Truncate the volume instead of rounding it because it's better# to trade too
        # little than too much.
        volume = volume.round_to_decimal_places(2, rounding=cdecimal.ROUND_DOWN)

        volume_str = '%.2f' % volume.amount
        price_str = '%.2f' % price.amount

```

Suggested fix, add for 8 decimal places:

```
        # Truncate the volume instead of rounding it because it's better# to trade too
        # little than too much.
        volume = volume.round_to_decimal_places(8, rounding=cdecimal.ROUND_DOWN)

        volume_str = '%.8f' % volume.amount
        price_str = '%.2f' % price.amount
```
Changes:
 '%.2f' to '%.8f'
'round_to_decimal_places(2,' .... to 'round_to_decimal_places(8,'

**Comment by garethdmm** [comment]
Good catch, this has been recently rejiggered in gemini and bitstamp so I'll probably just copy-past that over when I get a moment.

**Comment by itali43** [comment]
submitted a pull request fixing the error 🚀

**Comment by garethdmm** [comment]
Fixed #79
